src/css/reviews.css

```diff
@@ -0,0 +1,152 @@
+.reviews {
+    padding-top: 0;
+}
+
+.reviews .container {
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+}
+
+@media only screen and (min-width: 768px) {
+   .reviews .container {
+        gap: 64px;    
+  }
+}
+
+.reviews-title {
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 1.125;
+    letter-spacing: -0.02em;
+    text-align: center;
+    color: var(--black);
+}
+
+@media only screen and (min-width: 768px) {
+   .reviews-title {
+    font-size: 40px;
+    line-height: 1.2;
+  }
+}
+
+.reviews-description {
+    padding-top: 16px;
+    
+    font-size: 14px;
+    line-height: 1.71429;
+    font-weight: 400;
+    text-align: center;
+    color: rgba(18, 20, 23, 0.5);
+}
+
+@media only screen and (min-width: 768px) {
+   .reviews-description{
+       font-size: 18px;
+       line-height: 1.66667;
+       max-width: 647px;
+       margin: 0 auto;
+    }
+}
+
+
+.review-cards-wrapper {
+    display: flex;
+    justify-content: center;
+}
+
+@media only screen and (min-width: 768px) {
+    .review-cards-wrapper {
+        gap: 32px;
+    }
+}
+
+
+.review-card {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    padding: 20px;
+
+    border-radius: 16px;
+    max-width: 335px;
+    height: 278px;
+    background: var(--gray);
+}
+
+@media only screen and (min-width: 768px) {
+   .review-card {
+        gap: 32px;
+        width: calc((100% - 1*32px) / 2);
+        height: 326px;
+        padding: 32px 16px;
+  }
+}
+
+@media only screen and (min-width: 1280px) {
+   .review-card {
+        width: calc((100% - 2*32px) / 3);
+  }
+}
+
+@media only screen and (max-width: 768px) {
+    .card-2{
+        display:none
+    }
+}
+
+@media only screen and (max-width: 1279px) {
+    .card-3{
+        display: none;
+    }
```

Comment:
```
    review-cards-wrapper review-card:not(:first-child):not(:nth-child(2)) {
        display: none;
    }
```

Reply:
for hiding all except first and second card

Reply:
Hi! I have changed the code, could you please take a look?